doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -23,6 +23,7 @@
 * [Registrations](#registrations)
 * [`keyValues` not supported in `POST /v2/op/notify`](#keyvalues-not-supported-in-post-v2opnotify)
 * [Deprecated features](#deprecated-features)
+* [Additional fields in `httpInfo` for custom notifications](#additional-fields-in-`httpInfo`-for-custom-notifications)
```

Comment:
```suggestion
* [Additional fields in `httpInfo` for custom notifications](#additional-fields-in-httpInfo-for-custom-notifications)
```

Otherwise it seems the link doesn't work.

Reply:
Updated